src/System.Linq/src/System/Linq/Enumerable.cs

```diff
@@ -122,11 +122,6 @@ public IEnumerator<TSource> GetEnumerator()
 
             public abstract IEnumerable<TSource> Where(Func<TSource, bool> predicate);
 
-            public virtual TSource[] ToArray()
```

Comment:
Removing this from here was mainly to be able to have both Iterator and IPartition share the same interface-path to ToArray. Some Iterators have a ToArray that ends up going through Buffer twice, so just removing them and letting Buffer do what it would do anyway is a side-effect benefit.